### PR TITLE
added flymigration for db config and gitignore

### PR DESCRIPTION
**Pull request by saiharsha-eGov**

### CODE REVIEW

**Review by rushang7-eGov — COMMENTED** [review]
Instead of the lengthy name letter-of-indent-service, just use loi-service.

File: build/build-config.yml [hunk 1]
```diff
@@ -27,6 +27,14 @@ config:
       - work-dir: "estimate-service/src/main/resources/db"
         image-name: "estimate-service-db"
 
+  - name: "builds/digit-works/backend/LOI-Service"
+    build:
+      - work-dir: "backend/LOI-Service"
```
Comment (rushang7-eGov) [inline build/build-config.yml:32]: Keep this lowercase loi. We usually don't use uppercase in service names and directory names.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Intellij
+.idea/
```
Comment (rushang7-eGov) [inline .gitignore:2]: Don't add a gitignore at root level. Each project will have its own git ignore file.

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

File: build/build-config.yml (hunk 1 above)
Comment (saiharsha-eGov) [reply build/build-config.yml]: addressed the changes

File: .gitignore (hunk 2 above)
Comment (saiharsha-eGov) [reply .gitignore]: removed from root

**Review by rushang7-eGov — APPROVED** [review]

**Pull request merged**